### PR TITLE
Milestone 5. math-magicians project  Testing branch 

### PR DESCRIPTION
**Pull request by otmanTR**
In this branch we made the following changes:
- Added calculate.test.js and operate.test.js files
- Added Jest tests to calculate.js and operate.js files
- Added Home.test.js, Calcul.test.js, Calculator.test.js, Quote.test.js files
- Added App.test.js file
- Added React tests to Home.jsx, Calcul.jsx, Calculator.js, Quote.jsx
- All tests passed

### CODE REVIEW

File: src/tests/snapshots/Quote.test.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+import Quote from '../../pages/Quote';
+
+describe('Quote', () => {
+  test('should match Quote snapshot', () => {
+    const tree = render(
+      <Quote />,
+    );
+    expect(tree).toMatchSnapshot();
+  });
+});
```
Comment (danmainah) [inline src/tests/snapshots/Quote.test.js:12]: - [ ] @team great work on your components test however you forgot to add tests for the App component, kindly add it.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @otmanTR and @Oscah01 , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights

✔️ No linter errors
✔️ Gitflow was correctly used
✔️  Good design


# Required Changes ♻️

- [x] @otmanTR and @Oscah01 great work on your documentation, however, the title of your readme is wrong kindly change it.
- [x] @otmanTR and @Oscah01 kindly ensure your MIT link is working, also could be nice if the future features section design matches with other section designs.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md [hunk 2]
```diff
@@ -25,7 +25,7 @@
 
 
 
-# 📖 JavaScript Capstone Project <a name="about-project"></a>
+# 📖 Math Magicians Project <a name="about-project"></a>
```
Comment (TedLivist) [inline README.md:28]: - [ ] All components should be tested. Kindly add tests for the `src/App.js` component as requested by the previous reviewer.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @otmanTR and @Oscah01,

✔️ Good documentation for title and MIT as suggested

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (otmanTR) [reply README.md:28]: Thank you @TedLivist  but it was an optional suggestion. Because of this reason, I asked second opinion.

**Comment by medaminedev66** [comment]
Hello @otmanTR  👋,

_This is an additional review as a result of the second opinion request._

The first reviewer requested adding tests for the App component as a mandatory change. Since there is no [OPTIONAL] prefix next to the requested change, the requested change is mandatory.

As stated in the review summary, every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. The only signal for an optionally requested change is the [OPTIONAL] prefix.

Keep rocking 🚀!

**Review by Reem-lab — APPROVED** [review]
# Hi Team,👋🏻👋🏻

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🔥🔥💯💯



## Highlights 🔥🔥🔥🔥
- Followed Gitflow ✅
- No Linters error 💯✨
- Great working on the tests  🔥🔥


## Optional suggestions


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**